.markdownlint.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
You're not even running markdownlint anywhere... It's missing from `docs_and_scripts.yml`